### PR TITLE
Fix: Gantt chart unloading issue(fixes #245, #256)

### LINKED ISSUE #245: Gantt Chart - Label and Icons Reposition Incorrectly When Unloading Data
**Describe the bug**
In the Gantt chart, when multiple data sets are loaded, and then one data set is unloaded, the labels and icons do not reposition correctly. Resizing the browser enough can fix this issue, or unloading both data sets and then reloading them again.

**Link to CodeSandbox**

[Code Sandbox](https://codesandbox.io/s/cold-dust-y9ifk). Toggle both data sets on, then toggle one off to reproduce the issue. 

**Expected behavior**
I expect the labels and icons to reposition correctly, vertically centered in the remaining space.

**Screenshots**
Screen Shot 2020-06-22 at 2 49 11 PM

**Desktop (please complete the following information):**

-   OS: macOS Catalina 10.15.5
-   Browser: Chrome 83.0.4103.106 (Official Build) (64-bit)
--

**Additional context**

It appears this functioned correctly in the 1.x version which was the default for the Code Sandbox link in the issue template, so I was a bit confused at first. The issue definitely appears in the latest version, however.

### PR DESCRIPTION
**Pull request by AshishMotanamGurunadham**
**What is the purpose of this pull request? (Check any that's applicable)**

-   [ ] Feature
-   [x] Bug fix
-   [ ] Dependency updates
-   [ ] Documentation updates
-   [ ] Build related changes
-   [ ] Changes an existing functionality
-   [ ] Other, please explain:

**Does this introduce a breaking change?**

-   [ ] Yes
-   [x] No

**What changes did you make? (Give an overview)**

In the Gantt chart, when multiple data sets are loaded, and then one data set is unloaded, the labels and icons do not reposition correctly. Fixed that issue by making appropriate changes.

**Add any screenshots**

**Images after unloading a track from gantt chart:**
**Before fixing issue**

image

image

**After fixing issue**

image

image

### CODE REVIEW

File: src/main/js/controls/Gantt/Track.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ import { loadActions, unloadActions } from "./helpers/actionHelpers";
 import { loadActivities, unloadActivities } from "./helpers/activityHelpers";
 import {
     createTrackContainer,
-    removeTrackContainer,
+    removeTrackContainer, scaleGraph,
```
Comment (Veera81) [inline src/main/js/controls/Gantt/Track.js:19]: move the `scalegraph` next line

File: src/main/js/controls/Gantt/Track.js [hunk 2]
```diff
@@ -220,6 +220,7 @@ class Track extends GraphContent {
      * @inheritdoc
      */
     resize(graph) {
+        scaleGraph(graph.scale, graph.config);
```
Comment (Veera81) [inline src/main/js/controls/Gantt/Track.js:223]: Any unit tests for the change?

Comment (AshishMotanamGurunadham) [reply src/main/js/controls/Gantt/Track.js:223]: No veera. Test cases regarding scaling is already written veera.

Comment (sdadn) [reply src/main/js/controls/Gantt/Track.js:219]: Should the test cases be updated to ensure this bug does not occur?

File: src/main/js/controls/Gantt/Track.js (hunk 1 above)
Comment (AshishMotanamGurunadham) [reply src/main/js/controls/Gantt/Track.js]: [fixed](https://github.com/cerner/carbon-graphs/pull/263/commits/c0b2d8703781d7f0ae39f2330af0eed4365b8141#diff-bdf6ac5b711158c299d4a3c2dce8913fR20)

File: src/main/js/controls/Gantt/Track.js (hunk 2 above)
Comment (AshishMotanamGurunadham) [reply src/main/js/controls/Gantt/Track.js:219]: Added test cases to check y-co ordinate position and graph scale domain after unloading track. https://github.com/cerner/carbon-graphs/pull/263/commits/915542d71c0d77ba712b18fe3bef681f8dd79878

**Comment by Matthematic** [comment]
@AshishMotanamGurunadham Once this gets merged, how soon can we expect a release?

**Comment by Veera81** [comment]
@Matthematic Once it is merged, It will automatically release the carbon-graphs

**Pull request merged**